src/main/scala/strawman/collection/immutable/LazyList.scala

```diff
@@ -34,6 +35,10 @@ class LazyList[+A](expr: => LazyList.Evaluated[A])
         case Some((hd, tl)) => s"$hd #:: $tl"
       }
     else "LazyList(?)"
+
+  // HACK Needed because we inherit from two implementations (one from LinearSeq and one from SeqLikeFromIterable)
+  override def drop(n: Int): LazyList[A] = super.drop(n)
```

Comment:
Should `LinearSeq` have implementations at all or only be a marker trait with a subtype of `SeqLikeFromIterable` providing the implementations?